### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by delphiactual**
First pass at adding i18n to the filters.html...
This is just for the descriptions and does not apply to the filters themselves.

Still need to add i18n for vendors/activities/release info

### CODE REVIEW

**Comment by delphiactual** [comment]
After #1769 the italics can be used again for vendors/releases/activities again...

**Comment by bhollis** [comment]
Hmm, a lot of that text is in the vendors definitions, though, right? Why should we maintain them ourselves?

**Comment by delphiactual** [comment]
I thought I put my explanation in the second message... It was so we didn't need to load definitions into the dimSettingsCtrl...

**Comment by bhollis** [comment]
That's fair - though filters should probably have their own controller, and that can load definitions if it wants.

**Comment by delphiactual** [comment]
I'll see if I can figure out how to make a controller for filters...

**Comment by bhollis** [comment]
The whole filters page should just be its own component (with its own controller). I'd like to move those things out of popups for mobile.

**Comment by kyleshay** [comment]
I was wondering if we could make the filter modal a dropdown, and then for things like burns instead of listing each one, just showing 1 line item with a (multi?) select box for each option, that on change updates the search box, would be good for desktop as well as mobile

**Comment by bhollis** [comment]
Linking #1876

File: src/i18n/dim_en.json [hunk 1]
```diff
@@ -138,7 +138,40 @@
...
+        "CQ": "Crucible Quartermaster",
+        "EV": "Eververse",
+        "Gunsmith": "Gunsmith"
+      },
+      "Releases": {
+        "Vanilla": "Vanilla",
+        "tTK": "The Taken King",
+        "RoI": "Rise of Iron"
+      },
+      "Activities": {
```
Comment (kyleshay) [inline src/i18n/dim_en.json]: oooh

Comment (delphiactual) [inline src/i18n/dim_en.json]: ?

Comment (kyleshay) [inline src/i18n/dim_en.json]: didn't realize you had found those. will be useful for that page.

Comment (delphiactual) [inline src/i18n/dim_en.json]: yeah scoured bungie.net for them, the manifest was not helpful

File: src/i18n/dim_en.json [hunk 2]
```diff
@@ -80,7 +80,100 @@
...
+        "Dismantle": "Shows items you wish to eventually dismantle.",
+        "Infuse": "Shows items you wish to eventually infuse."
+      },
+      "Notes": "Search for items that you have tagged with custom notes",
+      "Rated": "Shows all weapons that have a rating from Destiny Tracker.",
+      "Rating": "Shows weapons based on their rating from Destiny Tracker.",
+      "Year": "Shows items from which year of Destiny they appeared in.",
+      "InLoadout": "Shows items that are included in any loadout.",
+      "Postmaster": "Items that are currently in the Postmaster.",
+      "Vendor": "Item is available from vendor <i>{{vendor}}</i> {{name}}",
```
Comment (delphiactual) [inline src/i18n/dim_en.json]: i18next can compile html :heart:

**Comment by delphiactual** [comment]
@bhollis I think this is good to go, I cannot figure out how to make a simple controller for this currently...

**Pull request merged**